flocker/acceptance/test_environment_variables.py

```diff
@@ -247,12 +245,15 @@ def get_mysql_node_2(ignored):
             Move MySQL to ``node_2`` and return a ``Deferred`` which fires
             with a connection to the previously created database on ``node_2``.
             """
+            # Listen on different port so it's clear we're connecting to
```

Comment:
Well, there's *two* things we want to assert:

1. External behaviour from point of view of user (data moved, ports are open, etc.) is as expected. We do this by testing it directly.
2. Flocker's understanding of the state of the world is correct. We do this by looking at state in control service.

Checking what Docker thinks the state of the world is doesn't really address either requirement.